### PR TITLE
updated pod number

### LINKED ISSUE #13: Add the pod!
Currently, then repository has members of the MLH team on it, but you should replace them with members of the pod.

You can update information in `_data/fellows.yml`

You should work on:
- [ ] Replacing the images
- [ ] Update the names and locations of each fellow
- [ ] Update the pod number inside of `_config.yml`

### PR DESCRIPTION
**Pull request by jaygala223**
## Please, go through these steps before you submit a PR.

- [X] My Pod Leader knows I'm working on this Pull Request
- [X] I've explained what the Pull Request is adding.
- [X] I've explained why this is important.

### CODE REVIEW

**Comment by jaygala223** [comment]
Hi @akshgpt7, can you please review this PR?

File: Gemfile.lock [hunk 1]
```diff
@@ -37,7 +37,7 @@ GEM
       rexml
     kramdown-parser-gfm (1.1.0)
       kramdown (~> 2.0)
-    liquid (4.0.3)
+    liquid (4.0.4)
```
Comment (akshgpt7) [inline Gemfile.lock]: Hey, you don't need to commit the Gemfile.lock. Can you omit that file from the PR?

File: _config.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-title: "Pod 22.PX.X"
+title: "Pod 23.APR.1"
```
Comment (akshgpt7) [inline _config.yml]: The exact pod name would be "23.APR.PREP.1" :)
(Sorry I missed your question in the issue comments)

**Review by akshgpt7 — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a couple of small tweaks and it's good to go.

File: _config.yml (hunk 2 above)
Comment (jaygala223) [reply _config.yml]: Oh right. On it!

File: Gemfile.lock (hunk 1 above)
Comment (jaygala223) [reply Gemfile.lock]: Yes sorry I didn't see this. I'll omit this file

**Comment by akshgpt7** [comment]
LGTM!

**Pull request merged**